game.py

```diff
@@ -1,3 +1,4 @@
+from projectiles import Projectile
 import pygame
```

Comment:
It would be better if it's the first import to have all internal module bundled at the same place.